src/collective/easyform/browser/fields.py

```diff
@@ -176,35 +220,35 @@ def __call__(self):
                 root = etree.fromstring(source, parser=parser)
             except etree.XMLSyntaxError as e:
                 IStatusMessage(self.request).addStatusMessage(
-                    "XMLSyntaxError: {0}".format(html.escape(safe_unicode(e.args[0]))),
+                    f"XMLSyntaxError: {html.escape(safe_text(e.args[0]))}",
```

Comment:
safe_text was not imported here so this f-string gives a traceback when it is called.

Reply:
Oops. thanks for fixing